packages/gestalt/src/PopoverEducational.js

```diff
@@ -166,10 +171,11 @@ export default function PopoverEducational({
         onDismiss={onDismiss}
         positionRelativeToAnchor
         showCaret
+        shouldFocus={shouldFocus ?? false}
         role={primaryAction && !children ? 'dialog' : role}
         size={size}
       >
-        {children ?? message ? (
+        {!children && message ? (
```

Comment:
This is the right logic, see snapshot change.

Reply:
That's not what I was suggesting. You don't need to reference `children` in the markup twice. The logic is saying:
- if there's `children`, render that
- else if there isn't children:
  - if there's `message`, render that with some other markup
  - else render null

So in code, we can express that as
```js
{children ?? message ? {markup} : null}
```
There is no need for an additional `children && null` line. This makes the relationship between `children` and `message` very clear, and eliminates any possibility for bugs where a reference to `children` gets changed and we render both `message` _and_ `children`.

Reply:
@dangerismycat We don't control the padding in children (as there are many existing instances) so if there's children, there can't be message+primaryAction. Children and message are exclusive.

Reply:
But it doesn't work 
<img width="202" alt="Screenshot 2023-03-01 at 12 43 44 AM" src="https://user-images.githubusercontent.com/10593890/222055209-72cb149b-2298-4073-8137-6ee8c8c490aa.png">

Reply:
It needs parethneses
<img width="292" alt="Screenshot 2023-03-01 at 12 44 58 AM" src="https://user-images.githubusercontent.com/10593890/222055348-db873508-e711-453a-b683-536b204e828d.png">